package-system/python/win_x64/open3d_python.patch

```diff
@@ -1,15 +1,108 @@
+diff --git a/Lib/ensurepip/__init__.py b/Lib/ensurepip/__init__.py
+index 94d40b0..fea685b 100644
+--- a/Lib/ensurepip/__init__.py
++++ b/Lib/ensurepip/__init__.py
+@@ -11,11 +11,10 @@ __all__ = ["version", "bootstrap"]
+ 
+ _SETUPTOOLS_VERSION = "47.1.0"
+ 
+-_PIP_VERSION = "20.1.1"
++_PIP_VERSION = "22.0.3"
+ 
+ _PROJECTS = [
+-    ("setuptools", _SETUPTOOLS_VERSION, "py3"),
+-    ("pip", _PIP_VERSION, "py2.py3"),
++    ("setuptools", _SETUPTOOLS_VERSION, "py3")
+ ]
+ 
+ 
+diff --git a/Mac/BuildScript/build-installer.py b/Mac/BuildScript/build-installer.py
+index 4fab488..e8a0f5a 100755
+--- a/Mac/BuildScript/build-installer.py
++++ b/Mac/BuildScript/build-installer.py
+@@ -331,7 +331,7 @@ def library_recipes():
+     if getDeptargetTuple() < (10, 5):
+         result.extend([
+           dict(
+-              name="Bzip2 1.0.6",
++              name="Bzip2 1.0.8",
+               url="http://bzip.org/1.0.6/bzip2-1.0.6.tar.gz",
```

Comment:
url appears to be 1.0.6 version, should be 1.0.8 version

Reply:
It turns out this file doesnt need patching at all. This is the patch for a windows build of python, but this section was patching the mac scripts, which will never be called anyways. Will remove this section